app/assets/javascripts/discourse/app/lib/load-morphlex.js

```diff
@@ -0,0 +1,3 @@
+export default async function loadMorphlex() {
```

Comment:
So future travellers know why it's here
```suggestion
/**
Plugins & themes are unable to async-import npm modules directly.
This wrapper provides them with a way to use morphlex, while keeping the `import()` in core's codebase.
*/
export default async function loadMorphlex() {
```